accounts/erd1qqqqqqqqqqqqqpgq9l2rc6wx6929npnn079z7tst4u5zlmaylzmsqt5n3h.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "MultiFights: NFT Players Staking",
+  "description": "",
+  "social": {
+    "website": "https://www.multi-fights.com",
+    "blog": "https://medium.com/@multifights",
+    "twitter": "https://twitter.com/MultiFights",
+    "whitepaper": "https://whitepaper.multi-fights.com/overview/introduction",
+    "discord": "https://discord.gg/du8urb6pd6"
+  },
+  "tags": ["multifights", "NFT", "staking"]
```

Comment:
use lowercase for NFT

Reply:
also use 
"tags": 
["multifights",
 "nft",
 "staking"
]

instead
"tags": ["multifights", "NFT", "staking"]